integration_tests/e2e/login.cy.ts

```diff
@@ -1,4 +1,4 @@
-import IndexPage from '../pages/index'
```

Comment:
`IndexPage` was our dummy stub page, which then turned into our Prisoner List Page.
I've renamed it to `PrisonerListPage`